README.md

```diff
@@ -63,3 +70,5 @@ Participation in the Kubernetes community is governed by the
 [security-model]: https://gateway-api.sigs.k8s.io/concepts/security-model
 [gh_release]: https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.0.0
 [godoc]: https://pkg.go.dev/sigs.k8s.io/gateway-api
+[conformance-docs]: ./conformance/README.md
```

Comment:
How does this relate to: https://github.com/kubernetes-sigs/gateway-api/blob/main/site-src/concepts/conformance.md#3-conformance-tests?

Shouldn't the README point to it, and add any new stuff there? Or are you planning to remove the concepts doc, and just point to this new README from there?

Reply:
+1. We may need to solve https://github.com/kubernetes-sigs/gateway-api/issues/2851 before continuing the work here, otherwise, we'll have to update the documentation with code and functions that will inevitably change. WDYT @shaneutt?

Reply:
Yep sounds good :+1:

Reply:
Ok, I updated the documentation in the website and pointed there from our main.

Reply:
/cc @shaneutt @frankbu